src/main/resources/html-rules.json

```diff
@@ -1,7 +1,14 @@
 {
   "attributes": {
     "span": {
-      "required": ["lang"]
+      "optional": [
+        [
+          "lang"
```

Comment:
Kanskje vurdere å implementere noe funksjonalitet for å ha minst en av attributene e.l?
Nå kan vel tomme `span`'s komme forbi validering